packages/importer/test/importer.test.cjs

```
@@ -4,9 +4,12 @@ const { test } = require("node:test")
 const { version } = require("../package.json")
 const { OneSchemaImporterClass } = require("../dist/main.js")
 
-function createImporter() {
+function createImporter(params) {
+  const listeners = []
   global.window = {
```

ESLint

[error] 'global' is not defined.

(no-undef)